internal/tfresource/retry.go

```diff
@@ -57,8 +57,8 @@ func RetryWhen(timeout time.Duration, f func() (interface{}, error), retryable R
 	return RetryWhenContext(context.Background(), timeout, f, retryable)
 }
 
-// RetryWhenAWSErrCodeEqualsContext retries the specified function when it returns one of the specified AWS error code.
-func RetryWhenAWSErrCodeEqualsContext(ctx context.Context, timeout time.Duration, f func() (interface{}, error), codes ...string) (interface{}, error) {
+// RetryWhenErrCodeEqualsContext retries the specified function when it returns one of the specified AWS error code.
```

Comment:
IMHO there is a (strong) case to be made for **not** renaming as this function (and its siblings) is explicitly checking that `error` is of type `*awserr.Error` in `tfawserr.ErrCodeEquals`.